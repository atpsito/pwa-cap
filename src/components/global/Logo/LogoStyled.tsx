import styled from "styled-components";

import LogoSVG from "../../../../public/assets/images/logo.svg";
import { LogoStyledProps as Props } from "./Logo.types";

const Logo = styled(LogoSVG)<Props>`
  height: 25vmin;
  pointer-events: none;
  padding-bottom: 3rem;

  @media (max-width: 450px) {
    height: 30vmin;
  }

  .Logo__circle {
    fill: var(--palette-primary);
  }
`;

export default Logo;
