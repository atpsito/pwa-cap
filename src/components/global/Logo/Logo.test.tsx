import React from "react";
import { render } from "@testing-library/react";

import LogoStyled from "./LogoStyled";

describe("Logo", () => {
  it("renders without crashing", () => {
    const { container } = render(<LogoStyled className="Logo" />);
    expect(container).toMatchSnapshot();
  });
});
