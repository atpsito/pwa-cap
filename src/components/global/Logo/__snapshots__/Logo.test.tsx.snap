// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders without crashing 1`] = `
.c0 {
  height: 25vmin;
  pointer-events: none;
  padding-bottom: 3rem;
}

.c0 .Logo__circle {
  fill: var(--palette-primary);
}

@media (max-width:450px) {
  .c0 {
    height: 30vmin;
  }
}

<div>
  <svg
    class="c0 Logo"
    id="Layer_1"
    viewBox="0 0 150 150"
    xmlns="http://www.w3.org/2000/svg"
  >
    <style>
      .st1
      {
      fill:#fff
      }
    </style>
    <circle
      class="Logo__circle"
      cx="75"
      cy="75"
      r="75"
    />
    <g
      id="XMLID_1_"
    >
      <path
        class="st1"
        d="M63.4 48.7V29.3h16.2v19.5h19.1v14h-19v34.7c0 6.3 3.2 8.9 8.7 8.9 3.6 0 6.8-.8 10.1-2.4v13.3c-4 2.3-8.6 3.6-14.6 3.6-11.9 0-20.4-5.2-20.4-20.8V48.7z"
        id="XMLID_6_"
      />
      <circle
        class="st1 logo__circle"
        cx="47.5"
        cy="55.5"
        id="XMLID_4_"
        r="8.1"
      />
    </g>
  </svg>
</div>
`;
